lib/modules/manager/maven-wrapper/__fixtures__/only-maven/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.5.4/apache-maven-3.5.4-bin.zip
```

Comment:
please inline those small fixtures

Reply:
will do

Reply:
please reduce noise, checkout https://github.com/renovatebot/renovate/blob/main/.github/contributing.md#resolve-review-comments-instead-of-commenting